tests/entity_classifier/test_data.py

```diff
@@ -0,0 +1,68 @@
+input_text1 = """
+Sachin's SSN is 222-85-4836.
+ITIN number 993-77 0690
+His AWS Access Key is: AKIAQIPT4PDORIRTV6PH.
+And Github Token is: ghp_hgu657yiujgwfrtigu3ver238765tyuhygvtrder6t7gyvhbuy5e676578976tyghy76578uygfyfgcyturtdf
+"""
+
+input_text2 = """
```

Comment:
Instead of `input_text1`, `input_text2`, etc, it would be if we can name them based on purpose of this input may be?

Reply:
As discussed, I'll update name wherever possible